README.pod

```diff
@@ -63,6 +63,27 @@ a subset of the functionality).
 It is based on Martin Berends' MiniDBI project, but unlike MiniDBI, DBDish
 aims to provide an interface that takes advantage of Perl 6 idioms
 
+=head2 Connecting to, and disconnecting from, a database
+
+For a database named I<hierarchy>, a connection may be established with any of
+the following syntaxes:
+
+    my ($dbh, $db);
+
+    $dbh = DBIish.connect("Pg", :database<hierarchy>);
+    $dbh.dispose;
+
+    $db = 'hierarchy';
+
+    $dbh = DBIish.connect("Pg", :database($db));
+    $dbh.dispose;
+
+    $dbh = DBIish.connect("Pg", database => $db);
+    $dbh.dispose;
+
+    $dbh = DBIish.connect("Pg", :$db);
```

Comment:
The "current" user is the current system user. My Linux user name is "nine", so libpq (PostgreSQL's client library) will connect to a database called "nine" if I don't give it any other database name:
nine@sphinx:~> perl -E 'use DBI; say DBI->connect("dbi:Pg:")->selectrow_array("select count(*) from cds");'
1